src/ClientApp/src/components/BohrungForm.js

```diff
@@ -315,9 +355,16 @@ export default function BohrungForm(props) {
               label="Y-Koordinate der Bohrung"
               type="number"
               variant="standard"
-              {...register("y_coordinate", { validate: validateYCoordinate })}
+              {...register("y_coordinate", {
+                validate: {
+                  range: (v) => validateYCoordinate(v) || "Die Y-Koordinate muss zwischen 1212000 und 1264000 liegen",
+                  distance: (v) =>
+                    validateDistance(v) ||
+                    "Die Distanz der neu erstellten Bohrung darf nicht mehr als 200 m zu den bereits vorhandenen Bohrungen betragen",
```

Comment:
PP: Die Einheit ohne Abstand neben die Zahl schreiben
```suggestion
                    "Die Distanz der neu erstellten Bohrung darf nicht mehr als 200m zu den bereits vorhandenen Bohrungen betragen",
```